src/register-org/containers/registered-address/registered-address.component.html

```diff
@@ -1,10 +1,11 @@
-<main id="content" class="govuk-main-wrapper" role="main" tabindex="-1">
+<main id="content" class="govuk-main-wrapper" role="main" tabindex="-1" #mainContent>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <!-- go back to previous page if not on international page, reset page otherwise -->
-      <a *ngIf="!startedInternational" class="govuk-back-link" href="javascript:history.back()">Back</a>
-      <a *ngIf="startedInternational" class="govuk-back-link"
+      <a *ngIf="!startedInternational && !addressChosen" class="govuk-back-link" href="javascript:history.back()">Back</a>
```

Comment:
@connorpgpmcelroy - I think we have to use the router link instead of history.back

Reply:
Ok, at time this was the consistent approach so will edit that. Makes it slightly easier to see what's happening within code i suppose